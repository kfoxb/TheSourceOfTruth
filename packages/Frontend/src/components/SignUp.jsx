import React from 'react';

export default function SignUp() {
  return (<h3>SignUp</h3>);
}
