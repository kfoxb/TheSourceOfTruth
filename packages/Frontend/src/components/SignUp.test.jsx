/* eslint-env jest */
import React from 'react';
import { shallow } from 'enzyme';
import SignUp from './SignUp';

describe('SignUp', () => {
  it('should render', () => {
    expect(shallow(<SignUp />)).toMatchSnapshot();
  });
});
