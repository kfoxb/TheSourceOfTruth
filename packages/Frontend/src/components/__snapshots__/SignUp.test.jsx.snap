// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignUp should render 1`] = `
<h3>
  SignUp
</h3>
`;
